homeassistant/components/landisgyr_heat_meter/coordinator.py

```diff
@@ -0,0 +1,36 @@
+"""Data update coordinator for the ultraheat api."""
+
+import logging
+
+import async_timeout
+import serial
+from ultraheat_api.response import HeatMeterResponse
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import POLLING_INTERVAL, ULTRAHEAT_TIMEOUT
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class UltraheatCoordinator(DataUpdateCoordinator):
```

Comment:
Can this not be typed better? Maybe this works?
```python
class UltraheatCoordinator(DataUpdateCoordinator[HeatMeterResponse]):
```

Reply:
Yes, thanks